CHANGELOG.md

```diff
@@ -2,12 +2,13 @@
 
 ## Unreleased
 
-- feat: Add empty sourcemaps wizard (#295)
-- feat: Add single tenant to self-hosted question (#277)
-- feat: Improve error handling of incorrect self-hosted URLs (#299)
 - feat(sourcemaps): Add Login and Project Selection flow (#300)
 - feat(sourcemaps): Add Sourcemaps as selectable integration (#302)
 - feat(sourcemaps): Add bundler selection prompt (#304)
```

Comment:
super-l:
Wasn't sure about alphabetization vs recency here but if we order by alphabet, let's change this too 
```suggestion
- feat(sourcemaps): Add bundler selection prompt (#304)
- feat(sourcemaps): Add Login and Project Selection flow (#300)
- feat(sourcemaps): Add Sourcemaps as selectable integration (#302)
```